auto_nag/scripts/regression.py

```diff
@@ -23,6 +24,16 @@ def columns(self):
     def sort_columns(self):
         return lambda p: (-p[2], -int(p[0]))
 
+    def bughandler(self, bug, data):
+        """We need to override bughandler from BZHandler because of this bug
```

Comment:
No need to override this one, but
```python
def handle_bug(self, bug, data):
    data[str(bug["id"])] = bug
```
should be enough.

Reply:
I override bughandler because I found BzCleaner bughandler behavior not as transparent as I would like it to be. Do you want me to update `BzCleaner.bughandler` to allow to returns the extra fields we need, like `type`?

Reply:
handle_bug is here for that: if you want add type, you can do something like:
```python
def handle_bug(self, bug, data):
    data[str(bug["id"])] = {"type": bug["bug_type"]}
    return data
```
and at the end you'll have id, summary and type available.
Let's say that a bug is uninteresting, you can then return None to skip it.